src/components/staking/PageValidator.vue

```diff
@@ -82,7 +79,7 @@
         <TmBtn
           id="undelegation-btn"
           class="undelegation-btn"
-          :disabled="delegation.amount === 0"
+          :disabled="delegation.noDelegation || session.addressRole === `stash`"
```

Comment:
nope! please allow and show in action modal

Reply:
oki